store/instance.go

```diff
@@ -559,6 +560,8 @@ func (s *Store) patchInstanceRaw(ctx context.Context, patch *InstancePatch) (*in
 
 // createInstanceImpl creates a new instance.
 func createInstanceImpl(ctx context.Context, tx *Tx, create *InstanceCreate) (*instanceRaw, error) {
+	// TODO(d): allow users to set resource_id.
+	resourceID := fmt.Sprintf("project-%s", uuid.New().String()[:8])
```

Comment:
instance-

I would consider to extra resource id prefix to a comment place.

Reply:
Done.

Let's keep it here for now. We can move this to v1 API surface later.